src/lib/y2storage/storage_manager.rb

```diff
@@ -192,13 +205,11 @@ def probe(probe_callbacks: nil)
     #   some errors (e.g., incomplete LVM VGs). This probed devicegraph
     #   is the result of sanitizing the initial raw probed.
     #
-    # @see #raw_probed
+    # @raise [Storage::Exception, Yast::AbortException] when probe fails
```

Comment:
ugh, that Storage::Exception is less expected here. At first it does not have global handler, so it can lead to many reports about internal error. I think we agreed only on AbortException with API change.

Reply:
Before these changes, such `Storage::Exception` were being masked because `#probe` simple catch them. But, most likely, next calls to the storage API would fail. We were discussing it yesterday, and that `Storage::Exception` are legitimate errors that YaST does not know how to recover from.

Reply:
well, if we do not know how to recover from it, would be option to just change it to AbortException?

Reply:
In that case, I would show a popup with the abort button.

Reply:
AbortException generally means the user has decided to abort. So I think it would be incorrect to turn any other unhandled exception into that.

Reply:
Letting the unhandled `Storage::Exception` to raise instead of masking it into a `return false` is actually consistent with our usual approach of "fail early".